src/java.desktop/share/classes/javax/swing/text/html/StyleSheet.java

```diff
@@ -3406,6 +3413,15 @@ boolean isW3CLengthUnits() {
         return w3cLengthUnits;
     }
 
+    /**
+     * Proxy value to compute inherited {@code font-size}.
+     *
+     * @see  <a href="https://www.w3.org/TR/CSS2/cascade.html">Assigning
+     *          property values, Cascading, and Inheritance</a>
+     */
+    static final Object FONT_SIZE_INHERIT =
+            new CSS().new FontSize().parseCssValue("100%");
```

Comment:
Do we need to instantiate new CSS object here? Can't we reuse "css" variable which is already initialized?

Reply:
I hadn't previously realized the `ViewAttributeSet` class is not static, so it should be possible to reuse the instance `css`.  Not sure if a separate instance per style sheet is necessary, but I'll prepare a revision.

Reply:
I have now made it use the instance `StyleSheet.css` variable.